bitnami/rabbitmq/templates/ingress.yaml

```diff
@@ -44,8 +44,14 @@ spec:
         - {{ .Values.ingress.hostname | quote }}
       secretName: {{ printf "%s-tls" .Values.ingress.hostname }}
     {{- end }}
-    {{- if .Values.ingress.extraTls }}
-    {{- toYaml .Values.ingress.extraTls | nindent 4 }}
+    {{- range .Values.ingress.extraTls }}
+    - hosts:
+      {{- range .hosts }}
+      - {{ include "common.tplvalues.render" ( dict "value" . "context" $ )}}
+      {{- end }}
+      {{- with .secretName }}
+      secretName: {{ include "common.tplvalues.render" ( dict "value" . "context" $ ) }}
+      {{- end }}
```

Comment:
Why do you want to breakdown this `extraTls` settings? Isn't it enough with something like:

```
    {{- include "common.tplvalues.render" ( dict "value" .Values.ingress.extraTls "context" $ ) | nindent 4 }}
```

as we do in the rest of the charts? What's your use case for this?

Reply:
It's a mistake, I forgot to revert the change when I have switch from "tpl" to custom "tplvalue" helper. You right, this change is not necessary, I revers to the default